### PR TITLE
HP-1888: Vps ip

### PR DESCRIPTION
**Pull request by bladeroot**
## Summary by CodeRabbit

- **New Features**
  - Added support for new VPS IP configurations: regular, public, and anycast.
  - Introduced new resource decorators for VPS IP configurations.

- **Updates**
  - Renamed 'proforma' document types to 'payment_request' across various views and models.
  - Updated translations for new service types in the Russian language.

- **Enhancements**
  - Improved the `convertAmount` method to handle additional VPS IP target types.
  - Enhanced `QuantityFormatterFactory` to support new cloud IP mappings.
  - Expanded `ResourceUnitWidget` to include new unit options for cloud IP configurations.

### CODE REVIEW

File: src/logic/bill/QuantityFormatterFactory.php [hunk 1]
```diff
@@ -39,6 +39,9 @@ final class QuantityFormatterFactory implements QuantityFormatterFactoryInterfac
         'referral'          => DefaultQuantityFormatter::class,
         'server_traf_max'   => DefaultQuantityFormatter::class,
         'vps_traf_max'      => DefaultQuantityFormatter::class,
+        'vps_ip_reqular'    => IPNumQuantity::class,
```
Comment (hiqsol) [inline src/logic/bill/QuantityFormatterFactory.php]: regular

**Review by hiqsol — APPROVED** [review]

File: src/models/decorators/target/VpsIpAnycastResourceDecorator.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Finance module for HiPanel
+ *
+ * @link      https://github.com/hiqdev/hipanel-module-finance
+ * @package   hipanel-module-finance
+ * @license   BSD-3-Clause
+ * @copyright Copyright (c) 2015-2019, HiQDev (http://hiqdev.com/)
+ */
+
```
Comment (SilverFire) [inline src/models/decorators/target/VpsIpAnycastResourceDecorator.php:10]: declare strict

File: src/models/decorators/target/VpsIpPublicResourceDecorator.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Finance module for HiPanel
+ *
+ * @link      https://github.com/hiqdev/hipanel-module-finance
+ * @package   hipanel-module-finance
+ * @license   BSD-3-Clause
+ * @copyright Copyright (c) 2015-2019, HiQDev (http://hiqdev.com/)
+ */
+
```
Comment (SilverFire) [inline src/models/decorators/target/VpsIpPublicResourceDecorator.php:10]: declare strict

File: src/models/decorators/target/VpsIpRegularResourceDecorator.php [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Finance module for HiPanel
+ *
+ * @link      https://github.com/hiqdev/hipanel-module-finance
+ * @package   hipanel-module-finance
+ * @license   BSD-3-Clause
+ * @copyright Copyright (c) 2015-2019, HiQDev (http://hiqdev.com/)
+ */
+
```
Comment (SilverFire) [inline src/models/decorators/target/VpsIpRegularResourceDecorator.php:10]: declare strict

**Review by SilverFire — CHANGES_REQUESTED** [review]

File: src/models/decorators/target/IpAnycastResourceDecorator.php [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<?php
+/**
```
Comment (SilverFire) [inline src/models/decorators/target/IpAnycastResourceDecorator.php:2]: declare strict

File: src/models/decorators/target/IpPublicResourceDecorator.php [hunk 6]
```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Finance module for HiPanel
+ *
+ * @link      https://github.com/hiqdev/hipanel-module-finance
+ * @package   hipanel-module-finance
+ * @license   BSD-3-Clause
+ * @copyright Copyright (c) 2015-2019, HiQDev (http://hiqdev.com/)
+ */
+
```
Comment (SilverFire) [inline src/models/decorators/target/IpPublicResourceDecorator.php:10]: declare strict

File: src/models/decorators/target/IpRegularResourceDecorator.php [hunk 7]
```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Finance module for HiPanel
+ *
+ * @link      https://github.com/hiqdev/hipanel-module-finance
+ * @package   hipanel-module-finance
+ * @license   BSD-3-Clause
+ * @copyright Copyright (c) 2015-2019, HiQDev (http://hiqdev.com/)
+ */
+
```
Comment (SilverFire) [inline src/models/decorators/target/IpRegularResourceDecorator.php:10]: declare strict

**Review by SilverFire — CHANGES_REQUESTED** [review]

**Review by SilverFire — APPROVED** [review]

**Pull request merged**